src/main/java/keyword/ToJson.java

```diff
@@ -21,15 +21,14 @@ public static void main(String[] args){
             JSONObject color1 = new JSONObject();
             JSONObject color2 = new JSONObject();
             JSONObject color3= new JSONObject();
-            JSONObject combine = new JSONObject();
             int key = 1;
-            String path = ".\\src\\keywords.txt";
+            String path = ".\\src\\main\\resource\\C_Keyword.txt";

```

Comment:
Even though this is the internal tool, it would be better to use args in main rather than hard code to the path

Reply:
This is just recommendation, not mandatory